tests/test_anyflowmodel.py

```
@@ -0,0 +1,436 @@
+    x_t = ns.forward_process(real, eps_noise, t)
+    dxt_dt = eps_noise - real
+
+    u_theta_jvp = model._jvp(x_t, t, r, dxt_dt, condition=cond)
+    assert torch.isfinite(u_theta_jvp).all(), "boundary samples must yield finite JVP estimates"
+
+
+def test_pretrain_consistency_bucket_pins_r_to_zero():
+    """With consistency_ratio=1.0 (and no flow-matching head), every sample's
+    r must be pinned to 0 (consistency to clean data, as in the reference)."""
+    model = _build_pretrain_model(consistency_ratio=1.0, r_sample_ratio=1.0)
+    t, r, is_diffusion = model._sample_t_r_buckets(4)
```

Ruff (0.15.15)

[warning] Unpacked variable `t` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)